apps/game_backend/priv/repo/migrations/20240509225708_add_quest_table.exs

```diff
@@ -0,0 +1,19 @@
+defmodule GameBackend.Repo.Migrations.AddQuestTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:quests) do
+      add :description, :string, default: ""
```

Comment:
Don't use `default: ""` if there is no string then `nil` should be the correct value. Having this `default` could cause a quest to be saved with `descriptor: ""` and then prevent modifications to other fields (not touching `descriptor`) due to our Ecto schema having `validate_required(..., [..., :descriptor, ...])`

Why do you wanted `default: ""`?

Reply:
This was force of habit, we don't need it at all